### PR TITLE
Tepdock/initial creation backend

### PR DESCRIPTION
**Pull request by tepdock**

### CODE REVIEW

File: BackendSecretProject/SecretProject/.idea/.idea.SecretProject/.idea/.gitignore
Comment (ShadowBLR28) [inline BackendSecretProject/SecretProject/.idea/.idea.SecretProject/.idea/.gitignore:1]: You should create an appropriate .gitignore file for .net app (or modify existing)

Comment (ShadowBLR28) [inline BackendSecretProject/SecretProject/.idea/.idea.SecretProject/.idea/.gitignore:1]: Why this file is located in the .idea folder?I think It should be located in the root folder of the repo

Comment (ShadowBLR28) [reply BackendSecretProject/SecretProject/.idea/.idea.SecretProject/.idea/.gitignore:1]: Please make dev as a target branch

Comment (ShadowBLR28) [reply BackendSecretProject/SecretProject/.idea/.idea.SecretProject/.idea/.gitignore:1]: Please rename project to 'SatoshiHero[SomeCommonNameLikeBackendOrApi]'

Comment (ShadowBLR28) [reply BackendSecretProject/SecretProject/.idea/.idea.SecretProject/.idea/.gitignore:1]: please merge main/dev branch

Comment (tepdock) [reply BackendSecretProject/SecretProject/.idea/.idea.SecretProject/.idea/.gitignore:1]: hi, cool, done it. how are u

Comment (tepdock) [reply BackendSecretProject/SecretProject/.idea/.idea.SecretProject/.idea/.gitignore:1]: hi, done it

**Review by ShadowBLR28 — APPROVED** [review]

**Pull request merged**